rust/src/action.rs

```diff
@@ -112,22 +112,25 @@ impl ColumnCountStat {
 
 /// Statistics associated with Add actions contained in the Delta log.
 #[derive(Serialize, Deserialize, Debug, Default)]
+#[serde(rename_all = "camelCase")]
 pub struct Stats {
     /// Number of records in the file associated with the log action.
-    pub numRecords: DeltaDataTypeLong,
+    pub num_records: DeltaDataTypeLong,
 
     // start of per column stats
     /// Contains a value smaller than all values present in the file for all columns.
-    pub minValues: HashMap<String, ColumnValueStat>,
+    pub min_values: HashMap<String, ColumnValueStat>,
     /// Contains a value larger than all values present in the file for all columns.
-    pub maxValues: HashMap<String, ColumnValueStat>,
+    pub max_values: HashMap<String, ColumnValueStat>,
     /// The number of null values for all columns.
-    pub nullCount: HashMap<String, ColumnCountStat>,
+    pub null_count: HashMap<String, ColumnCountStat>,
 }
 
 /// File stats parsed from raw parquet format.
 #[derive(Debug, Default)]
 pub struct StatsParsed {
+    // somehow #[serde(rename_all = "camelCase")] didn't j=wor just for this one..
```

Comment:
`j=wor` a typo?

Reply:
Yes... why I didn’t review line by line before committing...

so with this struct i had a problem - I could not add rename_all here, because compiler was complaining about it being out-of-scope. This is one of those rust features that are not fully loaded into my head yet.

I’ll look into it again later.